### ISSUE TITLE
Deleting Abstracts Generates PG::ForeignKeyViolation

### ISSUE THREAD

**Issue by tavisto** [obfuscurity/judy#26]
When trying to delete an Abstract the following error is reported

PG::ForeignKeyViolation: ERROR:  update or delete on table "abstracts" violates foreign key constraint "scores_abstract_id_fkey" on table "scores"
DETAIL:  Key (id)=(3) is still referenced from table "scores".

It appears that the models do not do a cascading delete.

**Comment by obfuscurity** [comment]
Yup, encountered this myself recently. Thanks for opening an issue for it.

Jason Dixon
